adaptors/adaptors.go

```diff
@@ -204,6 +205,7 @@ func (c *HwMgrAdaptorController) GetResourcePools(ctx context.Context, request i
 
 	resp, statusCode, err := adaptor.GetResourcePools(ctx, hwmgr)
 	if err != nil {
+		c.Logger.Error("unable to get resource pools from hardware manager %s: %w", request.HwMgrId, err)
```

Comment:
I don't think this is right. The first arg to the logger function is the log message, not a format. The additional fields are key-value pairs, which can be done with slog attribute functions like:
```suggestion
		c.Logger.Error("unable to get resource pools from hardware manager", slog.String("hwMgrId", request.HwMgrId), slog.String("error", err.Error()))
```